functions/src/index.ts

```diff
@@ -31,3 +31,49 @@ export const getSPLFunction = SPLFunction.getSPL;
 export const getBulletinFunction = bulletinFunction.getBulletin;
 export const getAndroidVersionFunction = androidVersionFunction.getAndroidVersion;
 
+//converter
+import * as data from './bulletin.json';
```

Comment:
as discussed, we should write this as a script that accepts this as an arg

Reply:
similarly the output path should be provided as well

Reply:
> similarly the output path should be provided as well, allow the caller to specify where it will be written

Hi Arjun, Evie has been integrating the code in this commit with the database code today. We just talked about these issues and she will address them in the next commit!